Tests/Fixtures/Generated/XCAssets/swift4-context-all-force-namespaces-false.swift

```diff
@@ -0,0 +1,134 @@
+#elseif os(iOS) || os(tvOS) || os(watchOS)
+  import UIKit.UIImage
+  internal typealias AssetColorTypeAlias = UIColor
+  internal typealias Image = UIImage
+#endif
+
+// swiftlint:disable superfluous_disable_command
+// swiftlint:disable file_length
+
+@available(*, deprecated, renamed: "ImageAsset")
+internal typealias AssetType = ImageAsset
+
+internal struct ImageAsset {
+  internal fileprivate(set) var name: String
+
+  internal var image: Image {
+    let bundle = Bundle(for: BundleToken.self)
+    #if os(iOS) || os(tvOS)
+    let image = Image(named: name, in: bundle, compatibleWith: nil)
+    #elseif os(OSX)
+    let image = bundle.image(forResource: NSImage.Name(name))
+    #elseif os(watchOS)
+    let image = Image(named: name)
+    #endif
+    guard let result = image else { fatalError("Unable to load image named \(name).") }
+    return result
+  }
+}
+
+internal struct ColorAsset {
+  internal fileprivate(set) var name: String
+
+  @available(iOS 11.0, tvOS 11.0, watchOS 4.0, OSX 10.13, *)
+  internal var color: AssetColorTypeAlias {
+    return AssetColorTypeAlias(asset: self)
+  }
+}
+
+// swiftlint:disable identifier_name line_length nesting type_body_length type_name
+internal enum Asset {
+  internal enum Colors {
+    internal enum _24Vision {
+      internal static let background = ColorAsset(name: "24Vision/Background")
+      internal static let primary = ColorAsset(name: "24Vision/Primary")
+    }
+    internal static let orange = ImageAsset(name: "Orange")
+    internal enum Vengo {
+      internal static let primary = ColorAsset(name: "Vengo/Primary")
+      internal static let tint = ColorAsset(name: "Vengo/Tint")
+    }
+    // swiftlint:disable trailing_comma
+    internal static let allColors: [ColorAsset] = [
+      _24Vision.background,
+      _24Vision.primary,
+      Vengo.primary,
+      Vengo.tint,
+    ]
+    internal static let allImages: [ImageAsset] = [
+      orange,
+    ]
+    // swiftlint:enable trailing_comma
+    @available(*, deprecated, renamed: "allImages")
+    internal static let allValues: [AssetType] = allImages
+  }
+  internal enum Images {
+    internal enum Exotic {
+      internal static let banana = ImageAsset(name: "Exotic/Banana")
+      internal static let mango = ImageAsset(name: "Exotic/Mango")
+    }
+    internal enum Round {
+      internal static let apricot = ImageAsset(name: "Round/Apricot")
+      internal static let apple = ImageAsset(name: "Round/Apple")
+      internal enum Double {
+        internal static let cherry = ImageAsset(name: "Round/Double/Cherry")
+      }
+      internal static let tomato = ImageAsset(name: "Round/Tomato")
+    }
+    internal static let `private` = ImageAsset(name: "private")
+    // swiftlint:disable trailing_comma
+    internal static let allColors: [ColorAsset] = [
+    ]
+    internal static let allImages: [ImageAsset] = [
+      Exotic.banana,
+      Exotic.mango,
+      Round.apricot,
+      Round.Red.apple,
```

Comment:
This fails to compile because there's no `Red` sub-enum. You still need to adapt the `allCases` bit of code in the template.

Reply:
Didn't see that one, i'll fix it